src/i18n/resources/en.js

```diff
@@ -2010,5 +2010,257 @@ The FRA team
       no19_area_available_for_public_recreation: 'Area available for public recreation 19',
       no20_area_available_for_public_recreation: 'Area available for public recreation 20',
     },
+
+    // Country Comments
+    countryComments: {
```

Comment:
Better to add country comments translations under panEuropean file with prefix panEuropean and import them from countryComments file

Reply:
I didn't catch what you exactly mean. Will see now in the daily